test-files/cases/hoot-js/highway-1.off/Config.conf

```diff
@@ -2,7 +2,7 @@
   "generic.line.match.threshold" : "0.6",
   "generic.line.miss.threshold" : "0.6",
   "generic.line.review.threshold" : "0.6",
-  "match.creators" : "hoot::ScriptMatchCreator,LineStringGeneric.js",
+  "match.creators" : "hoot::ScriptMatchCreator,Highway.js",
```

Comment:
I disabled generic highway and building conflation and tests. See PR description for more info.